features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Endpoints/Discover/index.js

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+'use strict';
+
+import React, {Component} from 'react'
+import {Link} from 'react-router-dom'
+import {Table, Icon, Dropdown, Button, Menu, message, Radio} from 'antd';
```

Comment:
please don't use antd modules. use material UI components

Reply:
I have started converting all my antd components to material UI in a separate branch. I will merge it as soon as it is complete